Lib/uuid.py

```diff
@@ -104,7 +114,7 @@ class UUID(object):
     """
 
     def __init__(self, hex=None, bytes=None, bytes_le=None, fields=None,
-                       int=None, version=None):
+                       int=None, version=None, is_safe=SafeUUID.unknown):
```

Comment:
Thanks for the review!  I did think about that, and it might indeed make sense, but it's a matter of style in this case, and I wasn't sure I wanted to introduce that to the API.  But on thinking about it again now that you bring it up, I does make sense.